prism/src/main/java/org/apache/falcon/resource/extensions/ExtensionManager.java

```diff
@@ -322,6 +354,112 @@ public APIResult validate(
         return new APIResult(APIResult.Status.SUCCEEDED, "Validated successfully");
     }
 
+
+    // Extension store related REST API's
+    @GET
+    @Path("enumerate")
+    @Produces({MediaType.APPLICATION_JSON})
+    public Response getExtensions() {
+        JSONArray results;
+
+        try {
+            List<String> extensions = STORE.getExtensions();
+            results = buildEnumerateResult(extensions);
+        } catch (StoreAccessException e) {
+            LOG.error("Enumerate extensions failed", e);
```

Comment:
Looks better to have a log message specific to this type of exception if you want to distinguish this exception from others, e.g. Failed when accessing extension store.